packages/gazelle_cli/lib/commands/delete/delete_project.dart

```diff
@@ -6,7 +6,7 @@ class DeletingProjectError {
   final String message;
 
   /// Creates a [DeletingProjectError].
-  const DeletingProjectError() : message = "Failed to delete project!";
+  const DeletingProjectError() : message = "No Gazelle project found.";
```

Comment:
I would change this class name to `DeleteProjectError`.

Reply:
DeletingProjectError is changed to DeleteProjectError